tokio/src/syscall/udp.rs

```diff
@@ -0,0 +1,336 @@
+use std::{
+    convert::TryFrom,
+    io, net,
+    task::{Context, Poll},
+};
+
+cfg_syscall! {
+    use crate::syscall::Syscalls;
+    use std::sync::Arc;
+    /// A representation of a UDP socket.
+    #[derive(Debug, Copy, Clone)]
```

Comment:
Little bit leery of putting a `Copy` impl in a public interface for this, since if we ever made it _not_ trivially `Copy`, removing the impl is a breaking change...but, this is in the `tokio-unstable` feature flag...

Reply:
I hadn't considered this, thank you!